multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -89,7 +131,15 @@ const makeTestContext = async (t: ExecutionContext) => {
     oracle: keys(oracleMnemonics).map(n => `${n}:${wallets[n]}`),
     usdcDenom,
     feedPolicy: JSON.stringify(makeFeedPolicyPartial(nobleAgoricChannelId)),
-    ...commonBuilderOpts,
+    // misconfigured a la https://github.com/Agoric/agoric-sdk/issues/11013
+    chainInfo: JSON.stringify(
+      replaceConnectionId(
+        JSON.parse(commonBuilderOpts.chainInfo),
+        'agoric',
+        'noble',
+        'connection-99999',
+      ),
+    ),
     assetInfo: fuAssetInfo(commonBuilderOpts.assetInfo),
```

Comment:
This builds the latest version of the contract. What I think we want for highest fidelity is something like the [downloadCoreEval](https://github.com/Agoric/agoric-sdk/blob/07f3dae8f68b087382e85bd827c7f1450b3b0088/packages/boot/test/fast-usdc/fast-usdc.test.ts#L72-L101) you recently added.  _With the caveat that `start-fast-usdc.js` will need to be amended to have different chain, asset and, connection info to match this environment._

But I think this approach works for now, since we don't have significant changes across versions and we're still simulating an incorrect counterparty connection id. I think you're on to the right approach with cbfc2753b3d922a911e917cfe38448857f04fce5